### ISSUE TITLE
[Inadyn] Backup size

### ISSUE THREAD

**Issue by lbouriez** [alexbelgium/hassio-addons#243]
Hello,
just checked the backup of Inadyn and it's very big, around 750mo.
![image](https://user-images.githubusercontent.com/4749766/157282227-08b4d447-38de-47c1-9e47-47d3b1d6ad97.png)
When going inside this is what I see:

- . is 760mo
  - image.tar is 760 mo
    - 
![image](https://user-images.githubusercontent.com/4749766/157282486-1f89734f-4228-4604-950f-b5c9817ccff5.png)

  - addon.json
  - apparmor.txt
- data is 204bytes

**Comment by lbouriez** [comment]
I actually noticed the same behaviour in the other addons (webtrees), backup was giant and again the same image.tar
I think it is because the images of the addons are not build and pushed into a registries so hassio is backuping the whole image...
Do you think it would be possible to add a github action to automatically build and push the docker images and so avoid all this giant backup size ?

**Comment by alexbelgium** [comment]
Hi, indeed it's the same for all addons autobuilding : the image is stored when doing a backup. 

The only way to avoid that is to enable autobuilds. However, I can't do that due to this bug in supervisor : https://github.com/home-assistant/supervisor/issues/3414. As long as this is not solved, i can't enable autobuilds or all my users won't be able to,update my addons.

So, if you want to help please like the supervisor issue so it is taken care off by the devs, allowing me to enable autobuilds and allowing all the nice things coming with it : less sd wear, faster updates, decrease backup size...

**Comment by lbouriez** [comment]
hoooo I see :/

**Comment by alexbelgium** [comment]
No, I can't duplicate everything, and the testing was already done : I'm just waiting for the supervisor bug being solved for implementation. So only new addons have the images enabled as it avoids the update bug. Here is the list of those addons : https://github.com/alexbelgium/hassio-addons/blob/master/.github/paths-filter.yml

**Comment by lbouriez** [comment]
Perfect so we can close the issues, waiting for the fixes I will just fork the addons :)
Thanks for your quick answers
